runtime/src/bank.rs

```diff
@@ -10065,22 +10077,15 @@ mod tests {
 
     #[test]
     fn test_debug_entrypoint() {
-        fn mock_process_instruction(
-            _program_id: &Pubkey,
-            _keyed_accounts: &[KeyedAccount],
-            _data: &[u8],
-        ) -> std::result::Result<(), InstructionError> {
-            Ok(())
-        }
-        fn mock_ix_processor(
+        fn mock_processor(
             _pubkey: &Pubkey,
             _ka: &[KeyedAccount],
             _data: &[u8],
-            _context: &mut dyn InvokeContext,
+            _invoke_context: &mut dyn InvokeContext,
         ) -> std::result::Result<(), InstructionError> {
             Ok(())
         }
-        assert!(!format!("{:?}", Entrypoint::Program(mock_process_instruction)).is_empty());
-        assert!(!format!("{:?}", Entrypoint::Loader(mock_ix_processor)).is_empty());
+        assert!(!format!("{:?}", Entrypoint::Program(mock_processor)).is_empty());
+        assert!(!format!("{:?}", Entrypoint::Loader(mock_processor)).is_empty());
```

Comment:
Do we need `Entrypoint::Loader` any longer?

Reply:
I guess only to add typing to distinguish a loader from a program.   I can try pulling it out and see what that does to the code though?

Reply:
Its originally there to handle the two entrypoints, now that there is one lets clean that up too

Reply:
@jackcmay - do you see any issue with deleting the `MessageProcessor::loaders` field entirely?   The only impact seems to be in `MessageProcessor::process_instruction`

Reply:
If loaders and programs have the same entrypoint signature now then we don't need to make the distinction between the two anymore.  So yeah, we can probably roll `loaders` into `programs`

Reply:
I just added b7d184c759754595aa560af76d5e5585c525f659 to merge them in MessageProcessor.  Once that passes CI, I'll make the larger changes need across the code base needed to de-Entrypoint

Reply:
Looks good, Entrypoint is now gone